post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java

```diff
@@ -151,11 +151,11 @@ public String getPathPrefixString() {
         return pathPrefixString;
     }
 
-    public List<Record> readRecordsFromFile(File recordsFile) throws IOException {
+    public List<Record> readRecordsFromFile(File recordsFile) {
         return attempt(() -> readFileAsString(getPathPrefixString() + recordsFile.getPath()))
                    .map(s -> JsonUtils.dtoObjectMapper.readValue(s, Record[].class))
                    .map(Arrays::asList)
-                   .orElse(failure -> List.<Record>of());
+                   .orElseThrow();
     }
```

Comment:
Here was the problem lying-> When we can not read string content into List of records -> returning empty List. From now on throwing exception when we can not deserialize string.